File: Resume/Scripts/models/responsibility.js
function Responsibility(responsibility) {
    var self = this;

    self.Id = ko.observable(responsibility.Id);
    self.Name = ko.observable(responsibility.Name);
    self.Description = ko.observable(responsibility.Description);
    self.PositionId = ko.observable(responsibility.PositionId);
    self.Percentage = ko.observable(responsibility.Percentage);
    self.dataify = ko.toJSON(self);
}

File: Resume/Models/Responsibility.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace Resume.Models
{
    public class Responsibility : Entity
    {
        [Required]
        public virtual string Name { get; set; }

        [Required]
        public virtual string Description { get; set; }

        [Required]
        public virtual double Percentage { get; set; }

        public virtual Position Position { get; set; }

        public virtual int PositionId { get; set; }

        public virtual ISet<Experience> Experiences { get; set; }

        public virtual IList<Tag> Tags { get; set; }

        public Responsibility()
        {
            Tags = new List<Tag>();
            Experiences = new HashSet<Experience>(); 
        }
    }
}